pkg/larker/loader/loader.go

```diff
@@ -146,11 +146,15 @@ func (loader *Loader) loadCirrusModule() (starlark.StringDict, error) {
 	}
 	result["base64"] = base64Module["base64"]
 
-	jsonModule, err := json.LoadModule()
-	if err != nil {
-		return nil, err
+	// Work around https://github.com/qri-io/starlib/pull/70
+	fixedJSONModule := &starlarkstruct.Module{
+		Name: "json",
+		Members: starlark.StringDict{
+			"loads": starlarkjson.Module.Members["decode"],
+			"dumps": starlarkjson.Module.Members["encode"],
```

Comment:
Should we also migrate to `decode`/`encode`?

Reply:
But why? The standard way is to use `json.dumps()` and `json.loads()` in Python, and re-using these names should make it more obvious.